Team2InitialUpdate

dantayy opened the pull request:
added rudimentary pieces to build upon for team 2's minigame

Commit:
Created structural folders for holding our assets, added manager game object to hold the manager script in, and added a rudimentary timing system to that manager script

Commit:
Update manager.cs

Added public var to easily change how long decision making rounds are on the fly in the editor

CavanVince committed:
Implemented a basic coin and player prefab.

CavanVince committed:
Added a Player Controller script that checks for one of three button presses to determine the player's action.

CavanVince committed:
Added a very basic "table" (background) for the player and coins to sit on.

Commit:
Move camera and add game objects

Commit:
Added public gameobject variables to the manager script and dropped the player prefabs in the scene into them so the manager can mess with them during the game loop

torirossini commented on GDDImagineGame/Assets/Team2/scripts/manager.cs:
```diff
@@ -7,7 +7,9 @@ public class manager : MonoBehaviour
     private float timer = 0.0f;
-
+    //public var for how long a "round" should be (time when players can choose an action)
+    public float timeToChoose = 0.0f;
```
If you don't intend for this variable to be used in other classes, you probably want to make it private and expose it to the editor using the tag [SerializeField].

Protecc the data :)

Commit:
Implemented rudimentary game loop with debug logs denoting player actions and score updates

Commit:
Began implementing functionality for player animation and UI

Commit:
Replaced TextMeshPro with basic Text

Commit:
General clean up

Commit:
Implemented scaling coin "piles" for all of the players as well as the main pot to indicate generally who's winning visually

Commit:
Rename manager.cs to Manager.cs

Commit:
Rename manager.cs.meta to Manager.cs.meta

Commit:
Merge pull request #10 from ComputerScienceHouse/team2coinpile

Team2CoinPile

CavanVince committed:
Implemented Rough UI to Team 2 Game

Went through old game logic and cleaned certain parts. Removed the ability to steal from the person across from you. Also implemented a rough UI system for scores.

CavanVince committed:
Gave players a starting score of 10 points.

Adjusted player starting score as well as fixed some score calculation logic when stealing.

CavanVince committed:
Implemented different game states.

Implemented a round number UI and also ready text for a better UX. Still need art for animations and cleaner UI.
